prow/spyglass/lenses/buildlog/buildlog.ts

```diff
@@ -137,7 +183,9 @@ async function loadLine(artifact: string, line: number): Promise<boolean> {
   const showers = document.querySelectorAll<HTMLDivElement>(`.show-skipped[data-artifact="${artifact}"]`);
   for (const shower of Array.from(showers)) {
     if (line >= Number(shower.dataset.startLine) && line < Number(shower.dataset.endLine)) {
-      await replaceElementWithContent(shower);
+      // TODO(fejta): could maybe do something smarter here than the whole
+      // block.
+      await replaceElementWithContent(shower, 0, 0);
```

Comment:
I think it's possible to use a details element, if you're okay loading the contents asynchronously or on load? (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) But not sure that quite fits with what's happening, and also not sure where latency's coming from on the replace.

Reply:
Ack.

I intend to work in this area in subsequent PRs. The latency is coming from `replaceElementWithContent()` sending a callback request to deck to fetch the additional lines: `const content = await spyglass.request(JSON.stringify(r));`

Reply:
Sounds good!